src/ldp/http/SimpleResponseWriter.ts

```diff
@@ -30,7 +31,7 @@ export class SimpleResponseWriter extends ResponseWriter {
     } else {
       input.response.setHeader('location', input.result.identifier.path);
       if (input.result.body) {
-        const contentType = input.result.body.metadata.contentType ?? 'text/plain';
+        const contentType = input.result.body.metadata.get(CONTENT_TYPE)?.value ?? 'text/plain';
```

Comment:
Law of Demeter?

But also, we _still_ might want to expose `.contentType` as it is so common (and short-circuit it to what you have written above).

Reply:
The main problem is consistency vs ease of use. I don't think we can predict in advance which metadata is going to be needed so I don't think we can get around (sort of) accessing metadata like this? Can always make an exception for some (like content-type) of course.

Reply:
I would make an exception for the very common ones indeed (was also the idea of the architectural diagram).